lib/utils.ts

```diff
@@ -20,3 +21,20 @@ const orderedActions: NextAction[] = Object.values(NextAction)
 export function ord(nextAction: NextAction): number {
   return orderedActions.indexOf(nextAction)
 }
+
+// convert to title case and replace '_ 'with ' '
+export function prettifyOption(value: string): string {
+  // leave emails untouched
+  if (value.includes('@')) return value
+  const ACRONYMS = ['Tmua', 'Ug', 'Gcse', 'Xii']
+  return value
+    .split('_')
+    .map(_.capitalize)
```

Comment:
```suggestion
    .map(capitalize)
```

Reply:
Adjusted in 0ab3615d8bbd217d7b6e8a424da4612ad9896811